README.md

```diff
@@ -1 +1,11 @@
 # thestonestore
+
+### SMTP Mail Client Information:
+---
+Server: smtpjs.com
+`Security Token` --> `833f4581-77f3-4e38-a7f4-ef13dee3abb5`
```

Comment:
Why we are putting this details here? Looks like this is the secure token. We shouldn’t need to keep it here.

Reply:
To make sure we don't lose the security key.

Reply:
Is the readme file is secure?